test/sqllogictest/pg_catalog_proc.slt

```diff
@@ -15,8 +15,8 @@ FROM pg_catalog.pg_proc
 WHERE proname = 'substring'
 ORDER BY oid
 ----
-substring  20003  936  NULL  25
-substring  20003  937  NULL  25
+substring  20004  936  NULL  25
+substring  20004  937  NULL  25
```

Comment:
I just want to call this change out. These are the OIDS for the `pg_catalog` schema. OIDs are documented to not be stable across restarts so technically this is fine. The change happened because the order that `Transaction` iterates over schemas happens to be different than the order that the Stash iterates over schemas. Even though it's technically fine, it's a bit scary since these OIDs probably haven't changed in a very long time.